config/environments/development.rb

```diff
@@ -45,14 +45,15 @@
   config.assets.raise_runtime_errors = true
 
   # Action Mailer settings
-  config.action_mailer.delivery_method = :letter_opener_web
```

Comment:
en dev on garde `letter_opener_web`

Reply:
Ok, c'est noté. J'avais fait ca pour pouvoir tester en local